src/filedesc.c

```diff
@@ -81,6 +81,7 @@ static void foreach_open_fd(void(pf)(int, void *), void *arg)
 		if (n == 0 || n == 1 || n == 2 || n == dir_fd) {
 			continue;
 		}
+		fprintf(stderr, "foreach_open_fd processing %d\n", n);
```

Comment:
Just a hint. When I need to force CI to run again I make a "git commit --amend" and don't change anything. Git will generate a different commit hash and when you push it to github it will consider a new version of the commit.

Reply:
There is actually a "Restart build" button in travis if you have write access to the repo (which I believe all current contributors have).

Reply:
:man_facepalming:

Reply:
Actually this was a debugging message to actually help me debug.